src/test/java/com/google/api/codegen/testdata/discoveries/datastore.v1beta3.json.overrides

```diff
@@ -0,0 +1,5 @@
+{
```

Comment:
There are many more that are already checked in. This one is included because the old datastore version was removed.